File: app/src/main/java/com/bankex/pay/presentation/ui/importwallet/ImportWalletActivity.java
package com.bankex.pay.presentation.ui.importwallet;

import android.os.Bundle;
import android.support.annotation.Nullable;
import com.bankex.pay.R;
import com.bankex.pay.presentation.ui.base.BaseActivity;

/**
 * Container activity that holds import wallet fragment.
 */
public class ImportWalletActivity extends BaseActivity {
	@Override
	public void onCreate(@Nullable Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_import_wallet);

		getSupportFragmentManager()
				.beginTransaction()
				.replace(R.id.fragment_container, new ImportWalletFragment())
				.commit();
	}
}

File: app/src/main/kotlin/com/bankex/wallet/ui/importwallet/ImportWalletActivity.kt
package com.bankex.wallet.ui.importwallet

import android.content.Context
import android.content.Intent
import android.support.v4.app.Fragment
import com.bankex.wallet.ui.base.activity.SingleFragmentActivity
import com.bankex.wallet.ui.createwallet.CreateWalletFragment

class ImportWalletActivity : SingleFragmentActivity() {

    override fun createFragment(): Fragment = ImportWalletFragment.newInstance()

    companion object {
        fun makeIntent(context: Context) = Intent(context, ImportWalletActivity::class.java)
    }
}
